docs/integrations/rsyslog.rst

```diff
@@ -0,0 +1,261 @@
+
+Add rsyslog integration to service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This can be configured in the Aiven Console by navigating to the target
+service overview page and then scrolling down to the **Manage integrations**
+button.
+
+
+You should be able to select your previously configured Rsyslog service
+integration by clicking Use integration in the modal window.
+
+.. image:: /images/integrations/remote-syslog-service-integrations.png
+   :alt: The page that shows the integrations available for a service
+
+Alternately, with the Aiven Client, first you need the id of the
+endpoint previously created
+
+::
+
+   avn service integration-endpoint-list --project your-project
+   ENDPOINT_ID                           ENDPOINT_NAME   ENDPOINT_TYPE
+   ====================================  ==============  =============
+   618fb764-5832-4636-ba26-0d9857222cfd  example-syslog  rsyslog
+
+Then you can link the service to the endpoint
+
+::
+
+   avn service integration-create --project your-project \
+       -t rsyslog -s your-service \
+       -D 618fb764-5832-4636-ba26-0d9857222cfd
+
+Integrating with a third party syslog service
+---------------------------------------------
+
+All integrations can be configured using the Aiven Console or the Aiven
+CLI though the examples are easier to copy and paste in the CLI form.
+
+Coralogix
+~~~~~~~~~
+
+For `Coralogix <https://coralogix.com/>`_ integration you need to use a custom ``logline`` format
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d coralogix -t rsyslog \
+       -c server=syslogserver.coralogix.us -c port=5142 \
+       -c tls=false -c format=custom \
+       -c logline="{\"fields\": {\"private_key\":\"YOUR_CORALOGIX_KEY\",\"company_id\":\"YOUR_COMPANY_ID\",\"app_name\":\"%app-name%\",\"subsystem_name\":\"programname\"},\"message\": {\"message\":\"%msg%\",\"program_name\":\"%programname%\",\"pri_text\":\"%pri%\",\"hostname\":\"%HOSTNAME%\"}}"
+
+.. note:: ``tls`` needs to be set to ``false``.
+
+Depending on whether your account ends in ``.com``, ``.us`` or ``.in``, you will
+need to use one of the following Syslog Endpoints for ``server``:
+
+-  ``syslogserver.coralogix.com``
+
+-  ``syslogserver.coralogix.us``
+
+-  ``syslogserver.app.coralogix.in``
+
+
+Datadog
+~~~~~~~
+
+For `Datadog <https://www.datadoghq.com/>`_ integration you need to use a custom ``logline`` format
+
+::
+
+   avn service integration-endpoint-create --project your-project \
+       -d datadog -t rsyslog \
+       -c server=intake.logs.datadoghq.com -c port=10516 \
+       -c tls=true -c format=custom \
+       -c logline='DATADOG_API_KEY <%pri%>1 %timestamp:::date-rfc3339% %HOSTNAME% %app-name% - - - %msg%'
+
+.. Note::
+   If you want to use Datadog EU environment, the service address is:
+   
+   * ``server``: ``tcp-intake.logs.datadoghq.eu``
+   * ``port``: ``443``
+
+.. Ideally the following link would use a ``:doc:`` role - for instance:
```

Comment:
I'm not sure what this is doing here but I think we're removing the section anyway so perhaps it doesn't matter